BackofficeBundle/Tests/EventSubscriber/ContentTypeSubscriberTest.php

```diff
@@ -145,7 +161,7 @@ public function testPreSetDataWithNoDatas()
         );
 
         Phake::when($this->fieldType1)->getFieldId()->thenReturn($fieldId);
-        Phake::when($this->transaltionChoiceManager)->choose(Phake::anyParameters())->thenReturn($label);
+        Phake::when($this->translationChoiceManager)->choose(Phake::anyParameters())->thenReturn($label);
```

Comment:
In this test, you never verify that the valueTransformerManager is called, or that the contentAttribute receive the stringValue you want